src/components/Search/Search.jsx

```diff
@@ -13,7 +15,11 @@ class Search extends React.Component {
   }
   render() {
     const { data, pros, ...rest } = this.props;
-    return <SearchResult {...rest} players={data || []} pros={pros || []} />;
+    return (<div>
+      <Helmet title={`${strings.title_search} ${this.props.location.query.q}`} />
```

Comment:
The only potential problem with this is that the order may need to be rearranged for certain languages.  You might want something more generic like `${this.props.location.query.q} - ${strings.title_search}`
where title_search is "Search"